order_app/lib/order_screen.dart

```diff
@@ -109,12 +95,7 @@ class _OrderScreenState extends State<OrderScreen> {
           ElevatedButton(
             onPressed: () {
               widget.orderModel?.ordered = true;
-              widget.orderModel?.items;
               widget.orderBox.put(widget.orderModel!);
-              widget.orderBox.getAll();
-              for (var element in items) {
-                element.itemCount = 0;
-              }
```

Comment:
How do you set the itemCount's initial value to zero, if you delete this part of the code?